.github/workflows/release.yml

```diff
@@ -4,6 +4,12 @@ permissions: {}
 on:
   workflow_dispatch:
 
+# Required for OIDC.
+# See: https://docs.npmjs.com/trusted-publishers
+permissions:
+  id-token: write
+  contents: read
```

Comment:
Push this down to jobs>release>permissions (preferable, to minimize scope, and specifically to exclude ~yml~ slack integration) or push up to `permissions: {}` on line 2 (it's confusing to immediately overwrite permissions key. I actually don't know, and don't want to know, the YAML semantics here :-)

Reply:
Can we already remove `NODE_AUTH_TOKEN: ${{ secrets.TIGERBEETLE_NODE_PUBLISH_KEY }}` in this PR?